source/ConversationPanel.cpp

```diff
@@ -182,10 +182,15 @@ void ConversationPanel::Draw()
 		
 			Point center = point + it.Center();
 			Point size(WIDTH, it.Height());
+
+			auto zone = Rectangle::FromCorner(point, size);
+			// If the mouse is hovering over this choice then we need to highlight it.
+			if(zone.Contains(hoverPoint))
+				choice = index;
```

Comment:
Looking at this, it seems like we might "ignore" subsequent keyboard inputs if the user's mouse is/gets parked where a choice would get drawn.

We might need to track the hover point only when choices are displayed, and ignore it if keyup/keydown are issued after it was last set.

Reply:
Ah good point. Fixed